tools/contract-analyzer/go.mod

```diff
@@ -123,3 +123,5 @@ require (
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 	lukechampine.com/blake3 v1.1.7 // indirect
 )
+
+replace github.com/onflow/cadence => ../..
```

Comment:
Until we release a new version of Cadence with the changes that the contract analyzer will need, we can just use the local version of Cadence for the analyzer

Reply:
Once this PR is merged, can you please open a follow-up PR which removes this and updated to the latest commit of the `master` branch?

Reply:
Yea, I will once all of the linting rules have been migrated over. Just thought I'd do them in separate PRs so they are easier to review.